### PR TITLE
pandoc-citeproc 0.16.4

### PR DESCRIPTION
**Pull request by dfkotz**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/pandoc-citeproc.rb [hunk 1]
```diff
@@ -5,8 +5,8 @@ class PandocCiteproc < Formula
 
   desc "Library and executable for using citeproc with pandoc"
   homepage "https://github.com/jgm/pandoc-citeproc"
-  url "https://hackage.haskell.org/package/pandoc-citeproc-0.16.2/pandoc-citeproc-0.16.2.tar.gz"
```
Comment (SMillerDev) [inline Formula/pandoc-citeproc.rb]: It seems you changed the source when updating. Could you change it to be `https://hackage.haskell.org/package/pandoc-citeproc-0.16.3.1/pandoc-citeproc-0.16.3.1.tar.gz`?

**Comment by dfkotz** [comment]
Sorry, this is my first time using `brew bump-formula-pr`.

I just found a download link and used it for the `url` field in that command.

Do I need to start over, using the URL you suggest, or are you suggesting I edit a file?

dave


On Nov 18, 2019, at 2:06 PM, Sean Molenaar <notifications@github.com<mailto:notifications@github.com>> wrote:


@SMillerDev commented on this pull request.

________________________________

In Formula/pandoc-citeproc.rb<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FHomebrew%2Fhomebrew-core%2Fpull%2F46868%23discussion_r347369025&data=02%7C01%7CDavid.F.Kotz%40Dartmouth.edu%7C41f567fe67764cebaba708d76c2813e3%7C995b093648d640e5a31ebf689ec9446f%7C0%7C0%7C637096791702441056&sdata=nsq%2FXyfzzpy%2B9UJj8HVbRJ4%2BjQPHYqqK6nBB2iFDbco%3D&reserved=0>:

> @@ -5,8 +5,8 @@ class PandocCiteproc < Formula

   desc "Library and executable for using citeproc with pandoc"
   homepage "https://github.com/jgm/pandoc-citeproc"
-  url "https://hackage.haskell.org/package/pandoc-citeproc-0.16.2/pandoc-citeproc-0.16.2.tar.gz"


It seems you changed the source when updating. Could you change it to be https://hackage.haskell.org/package/pandoc-citeproc-0.16.3.1/pandoc-citeproc-0.16.3.1.tar.gz?

**Comment by SMillerDev** [comment]
> Do I need to start over, using the URL you suggest, or are you suggesting I edit a file?

You don't have to start over, you could edit the pandoc-citeproc formula file (maybe even in the github UI) and insert the link I send before instead of the URL you used

**Comment by dfkotz** [comment]
I made the edit - I think!

On Nov 18, 2019, at 2:22 PM, Sean Molenaar <notifications@github.com<mailto:notifications@github.com>> wrote:


Do I need to start over, using the URL you suggest, or are you suggesting I edit a file?

You don't have to start over, you could edit the pandoc-citeproc formula file (maybe even in the github UI) and insert the link I send before instead of the URL you used

**Comment by Bo98** [comment]
We've just updated Pandoc so let's see if this builds properly now.

@BrewTestBot test this please

**Comment by Bo98** [comment]
```
==> Downloading https://hackage.haskell.org/package/pandoc-citeproc-0.16.3.1/pandoc-citeproc-0.16.3.1.tar.gz
Error: An exception occurred within a child process:
  ChecksumMismatchError: SHA256 mismatch
Expected: 5fb4a131c48bfba2a3d6a0be9c5761d122b46bc161313466b698e22f20ac1ac2
  Actual: 5fb01bc1e523fcd862a03e7e616477701fb5d33ddbc0f7ba28ae1ecae4deda89
```

There's also a 0.16.4 now so maybe worth updating this for that.

**Comment by dfkotz** [comment]
Thanks for the update.  Are you asking me to take action?  I’m still not sure exactly how to do a ‘bump’ properly.

On Nov 24, 2019, at 9:28 PM, Bo Anderson <notifications@github.com<mailto:notifications@github.com>> wrote:


==> Downloading https://hackage.haskell.org/package/pandoc-citeproc-0.16.3.1/pandoc-citeproc-0.16.3.1.tar.gz
Error: An exception occurred within a child process:
  ChecksumMismatchError: SHA256 mismatch
Expected: 5fb4a131c48bfba2a3d6a0be9c5761d122b46bc161313466b698e22f20ac1ac2
  Actual: 5fb01bc1e523fcd862a03e7e616477701fb5d33ddbc0f7ba28ae1ecae4deda89


There's also a 0.16.4 now so maybe worth updating this for that.

**Comment by Bo98** [comment]
It seems like the sha256 supplied doesn't match the sha256 of the archive the CI downloaded.

I can fix that for you if you prefer.

**Comment by dfkotz** [comment]
Sure, but show me how to do it right next time...

On Nov 24, 2019, at 9:32 PM, Bo Anderson <notifications@github.com<mailto:notifications@github.com>> wrote:


It seems like the sha256 supplied doesn't match the sha256 of the archive the CI downloaded.

I can fix that for you if you prefer.

**Comment by Bo98** [comment]
`brew bump-formula-pr --url=<url> --sha256=<sha256> <formula>`. Exact ordering of the arguments doesn't really matter.

It looks like you did manage to change the url and sha256 but the new sha256 doesn't seem to be correct.

I'll update the pull request to fix change (and also update it to 0.16.4 since that was released since this pull request was opened).

**Comment by dfkotz** [comment]
Thanks.

for clarity, the url argument should be something like
https://hackage.haskell.org/package/pandoc-citeproc-0.16.3.1/pandoc-citeproc-0.16.3.1.tar.gz
and the sha56 argument should be the output of the sha256sum command, applied to that file?

dave

On Nov 24, 2019, at 9:40 PM, Bo Anderson <notifications@github.com<mailto:notifications@github.com>> wrote:


brew bump-formula-pr --url=<url> --sha256=<sha256> <formula>. Exact ordering of the arguments doesn't really matter.

It looks like you did manage to change the url and sha256 but the new sha256 doesn't seem to be correct.

I'll update the pull request to fix change (and also update it to 0.16.4 since that was released since this pull request was opened).

**Comment by Bo98** [comment]
Yeah, that should work.

**Comment by Bo98** [comment]
Looking back at the pull request, I imagine what happened was the sha256 of the file on the original URL you used was different than the one you changed it to later.

**Comment by Bo98** [comment]
Thank you for your first pull request @dfkotz! It is now merged.

**Comment by dfkotz** [comment]
Thank you!  I hope to be able to help with pull requests in the future.  I’ll keep this thread as notes.  I would also recommend finding someone to update the documentation to help other new newbies use ‘bump’ properly.

thanks,
dave


On Nov 24, 2019, at 11:43 PM, Bo Anderson <notifications@github.com<mailto:notifications@github.com>> wrote:


Thank you for your first pull request @dfkotz<https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fdfkotz&data=02%7C01%7CDavid.F.Kotz%40Dartmouth.edu%7Cb6fdbd3ae0ea4ae76a1708d7712fc6ff%7C995b093648d640e5a31ebf689ec9446f%7C0%7C0%7C637102322321697180&sdata=35Znx1%2Bilq2NhJ%2BC80y6cTSSIELmqgI5sM%2BtG3U51dU%3D&reserved=0>! It is now merged.

**Pull request closed without merging**